scapy/arch/linux.py

```diff
@@ -65,6 +65,17 @@
 RTF_UP = 0x0001  # Route usable
 RTF_REJECT = 0x0200
 
+# From if_packet.h
```

Comment:
They are the possible values for the packet direction (as present in sockaddr_ll struct).
This information is useful, for example, to understand if the sniffed packet is incoming or outgoing.